dbscripts/xml/ojs_schema.xml

```diff
@@ -63,7 +63,7 @@
 		<field name="setting_name" type="C2" size="255">
 			<NOTNULL/>
 		</field>
-		<field name="setting_value" type="X"/>
+		<field name="setting_value" type="XL"/>
```

Comment:
Are we hitting a size limit in practice?

Reply:
In our case, the type TEXT is NVARCHAR and X is 4000 while XL is MAX. A medium size text would fail to be saved in the database. It was a problem with the column type like you said a couple weeks ago. If it is not an issue for any other clients, we can manage it on our side to avoid changing the code for no real reason. I believe X can hold a huge amount of data in MySQL and PostgresSQL.

Reply:
@bozana, this is the discussion about column types for settings that I mentioned on Monday. The biggest concern about going to XL, I think, is queries that search that column for a certain value. Off the top of my head, the `getBySetting` functions (used e.g. for PID lookups) qualify. I don't think we have many else like this.